src/features/settings-panel/fontfamily-selector.tsx

```
@@ -14,17 +14,34 @@ import { SettingsPanelItem } from './ui/item';
+
+  const handleCardClick = () => {
+    // Programmatically click the select trigger to open it
+    selectRef.current?.click();
+  };
 
   return (
-    <SettingsPanelItem value={fontFamily}>
+    <SettingsPanelItem
+      value={fontFamily}
+      onClick={handleCardClick}
+      role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)